include/dcmqi/MultiframeConverter.h

```diff
@@ -35,15 +35,21 @@
 
 using namespace std;
 
+// common type definitions
 typedef short ShortPixelType;
 typedef itk::Image<ShortPixelType, 3> ShortImageType;
 typedef itk::ImageFileReader<ShortImageType> ShortReaderType;
 
 namespace dcmqi {
 
-  class ConverterBase {
+  class MultiframeConverter {
```

Comment:
MultiframeConverter should be MultiFrameConverter


ifdefs should be renamed according to class name
```
#ifndef DCMQI_CONVERTERBASE_H
#define DCMQI_CONVERTERBASE_H
```